src/test/java/step1/calculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,47 @@
+package step1.calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class StringAddCalculatorTest {
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
+
+        result = StringAddCalculator.splitAndSum("");
+        assertThat(result).isEqualTo(0);
+    }
+    @Test
```

Comment:
공백 라인을 추가해주면 좋을 것 같아요.
IDE로 `IntelliJ IDEA`를 사용하고 계신다면 `Ctrl + Alt + L` 단축키 활용을 추천드립니다.
```suggestion
    }
    
    @Test
```